app/client/components/contactUs/contactUsForm.tsx

```diff
@@ -70,8 +62,8 @@ export const ContactUsForm = (props: ContactUsFormProps) => {
       window.guardian?.identityDetails?.email) ||
       ""
   );
-  const [details, setDetails] = useState<string>("");
-  const [detailsRemainingCharacters, setDetailsRemainingCharacters] = useState<
+  const [message, setDetails] = useState<string>("");
```

Comment:
if we are renaming the state to `message` should we also rename the set state to `setMessage`?